ch2/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.9-slim
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    software-properties-common \
+    git \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
+RUN pip install -U pip &&\
+    pip install mlflow psycopg2-binary boto3
+RUN cd /tmp && \
+    wget https://dl.min.io/client/mc/release/linux-amd64/mc && \
+    chmod +x mc && \
+    mv mc /usr/bin/mc
```

```suggestion
FROM amd64/python:3.9-slim

RUN pip install -U pip &&\
    pip install mlflow psycopg2-binary boto3

RUN cd /tmp && \
    wget https://dl.min.io/client/mc/release/linux-amd64/mc && \
    chmod +x mc && \
    mv mc /usr/bin/mc
```

1. base image를 불러올 때 m1에서는 에러가 발생하는 것들이 있어서 amd64로 통일하면 좋을 것 같아요. database 챕터도 m1에서 하다가 에러가 나서 amd64로 바꿨어요.
2. `build-essential`, `software-properties-common`, `git`, `wget` 은 굳이 설치하지 않아도 될 것 같아요.
3. 중간중간에 띄어쓰기를 하면 좋을 것 같아요.

---

`git`, `wget` 은 추후 `docker-compose.yaml` 에서 `mlflow` 를 띄울때 사용됩니다! base image는 수정 하겠습니다~!

---

mlflow을 띄울 때 `git`과 `wget`이 어떻게 쓰이나요??

---

git은 `mlflow server` command에서 설치 되어 있지 않은 경우 warninig을 발생시키며
`wget`의 경우 현재 Dockerfile의 두번째 `RUN`에서
` wget https://dl.min.io/client/mc/release/linux-amd64/mc && \` 구문에서 사용 됩니다!

---

아 그러네요 알겠습니다! 나머지 1, 3번만 반영 부탁드릴게요~

---

반영 했습니다!